_layouts/robot.html

```diff
@@ -56,5 +58,56 @@
     </div>
     {% endif %}
 
-</div>
+    <div class="column is-6">
+        {% if page.Specifications %}
+        <div class="content">
+            <h2>Specifications</h2>
+            <ul>
+                {% for Specification in page.Specifications %}
+                <li>{{ Specification.item }}</li>
+                {% endfor %}
+            </ul>
+        </div>
+        {% endif %}
+        {% if page.Capabilities %}
+        <div class="content">
+            <h2>Capabilities</h2>
+            <ul>
+                {% for Capability in page.Capabilities %}
+                <li>{{ Capability.item }}</li>
+                {% endfor %}
+            </ul>
+        </div>
+        {% endif %}
+    </div>
 
+    <div class="column is-6">
+
+
+        {% if page.Competitions %}
+        <div class="content">
+            <h2>Competitions</h2>
+            <ul>
+                {% for Competition in page.Competitions %}
+                <li>{{ Competition.item }}</li>
+                {% endfor %}
+            </ul>
+        </div>
+        {% endif %}
+        {% if page.Capabilities %}
+        <div class="content">
+            <h2>Links</h2>
+            <ul>
+                <div style="display: flex; flex-direction: row;display: flex; justify-content: center; align-items: center;">
+
+                {% for Item in page.Links %}
+                    <a href="{{ Item.link }}"><div style="text-decoration: underline; font-size: 1.5em; margin-right: 2.5rem; display: flex; justify-content: center; align-items: center;width:7em;height:6em;background-color:#990202;border-radius:24%;color:#F2F2F2;">{{ Item.item }}</div></a>
```

Comment:
If we're going to use an unordered list (`<ul>`), then please wrap the link with a list item (`<li>`).

Reply:
https://github.com/FRCTeam1987/teambroncobots.com/pull/50/commits/f23594b3fcb32a5f7637a99288e87e3d1dd50c12